mixer/adapter/signalfx/registry.go

```diff
@@ -0,0 +1,191 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package signalfx
+
+import (
+	"container/list"
+	"sort"
+	"sync"
+	"time"
+
+	"github.com/signalfx/golib/datapoint"
+	"github.com/signalfx/golib/sfxclient"
+)
+
+type metricID string
+
+var currentTime = time.Now
```

Comment:
i don't think this matters much, but should this be part of the `registry` type? It appears to only be used within the context of `registry`?

Reply:
It exists solely to make unit testing easier by being able to fake out `time.Now`, but I'll move it into the registry struct to avoid a global variable, if nothing else.